src/lib.rs

```diff
@@ -1040,14 +1061,16 @@ impl MatchOptions {
     /// MatchOptions {
     ///     case_sensitive: true,
     ///     require_literal_separator: false,
-    ///     require_literal_leading_dot: false
+    ///     require_literal_leading_dot: false,
+    ///     follow_links: false,
     /// }
     /// ```
     pub fn new() -> Self {
         Self {
             case_sensitive: true,
             require_literal_separator: false,
             require_literal_leading_dot: false,
+            follow_links: false,
```

Comment:
To preserve the existing default behavior, I think it would be best to  default to `true` here.

Reply:
Done